pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -262,6 +262,7 @@ type ClusterRef TidbClusterRef
 
 type TidbMonitorStatus struct {
 	// Storage status for deployment
+	// Deprecated from v1.2.0
```

Comment:
This is a new field in v1.2.0, actually it will not show up if it's empty because of `omitempty`, so I think no need to add this.

Reply:
ok